homepage/tests/add_review_search_tests.py

```diff
@@ -42,15 +42,20 @@ def test_lexicography_order_get_courses_ordered_by_name():
 
 
 # --------Front End testing-------- #
+@pytest.fixture
+def sign_in(client):
+    client.post('/users/sign_in/', data={'username': 'testUser1', 'password': 'password123'})
+
+
 @pytest.mark.django_db
-def test_renders_add_review_search_template(client):
+def test_renders_add_review_search_template(client, sign_in):

```

Comment:
because you are not using the return value of sign_in in your test. 
please try to use @pytest.mark.usefixtures("sign_in") something like:
```suggestion
@pytest.mark.usefixtures("sign_in")
def test_renders_add_review_search_template(client):

```